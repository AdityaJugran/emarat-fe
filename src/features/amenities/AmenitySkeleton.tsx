import { Col, Skeleton } from 'antd';
import Card from 'features/shared/components/styledComponents/Card';

export default function AmenitySkeleton() {
	return (
		<>
			{[...Array(6)].map((e, i) => (
				/* eslint-disable react/no-array-index-key */
				<Col key={i} xs={22} sm={11} lg={8} className="w-full">
					<Card className="flex items-center">
						<Skeleton.Avatar className="my-7" size="large" active />
						<div className="pl-6 w-full">
							<Skeleton paragraph active />
						</div>
					</Card>
				</Col>
			))}
		</>
	);
}
