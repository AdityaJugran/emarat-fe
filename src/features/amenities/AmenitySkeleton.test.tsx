import AmenitySkeleton from 'features/amenities/AmenitySkeleton';
import renderer from 'react-test-renderer';

it('Amenity Skeleton', () => {
	const tree = renderer.create(<AmenitySkeleton />).toJSON();
	expect(tree).toMatchSnapshot();
});
