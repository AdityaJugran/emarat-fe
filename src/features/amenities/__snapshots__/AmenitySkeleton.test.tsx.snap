// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Amenity Skeleton 1`] = `
Array [
  <div
    className="ant-col w-full ant-col-xs-22 ant-col-sm-11 ant-col-lg-8"
    style={Object {}}
  >
    <div
      className="sc-bdfBQB DEPcs flex items-center"
    >
      <div
        className="ant-skeleton ant-skeleton-element ant-skeleton-active my-7"
      >
        <span
          className="ant-skeleton-avatar ant-skeleton-avatar-lg ant-skeleton-avatar-circle"
          style={Object {}}
        />
      </div>
      <div
        className="pl-6 w-full"
      >
        <div
          className="ant-skeleton ant-skeleton-active"
        >
          <div
            className="ant-skeleton-content"
          >
            <h3
              className="ant-skeleton-title"
              style={
                Object {
                  "width": "38%",
                }
              }
            />
            <ul
              className="ant-skeleton-paragraph"
            >
              <li
                style={
                  Object {
                    "width": undefined,
                  }
                }
              />
              <li
                style={
                  Object {
                    "width": undefined,
                  }
                }
              />
              <li
                style={
                  Object {
                    "width": "61%",
                  }
                }
              />
            </ul>
          </div>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="ant-col w-full ant-col-xs-22 ant-col-sm-11 ant-col-lg-8"
    style={Object {}}
  >
    <div
      className="sc-bdfBQB DEPcs flex items-center"
    >
      <div
        className="ant-skeleton ant-skeleton-element ant-skeleton-active my-7"
      >
        <span
          className="ant-skeleton-avatar ant-skeleton-avatar-lg ant-skeleton-avatar-circle"
          style={Object {}}
        />
      </div>
      <div
        className="pl-6 w-full"
      >
        <div
          className="ant-skeleton ant-skeleton-active"
        >
          <div
            className="ant-skeleton-content"
          >
            <h3
              className="ant-skeleton-title"
              style={
                Object {
                  "width": "38%",
                }
              }
            />
            <ul
              className="ant-skeleton-paragraph"
            >
              <li
                style={
                  Object {
                    "width": undefined,
                  }
                }
              />
              <li
                style={
                  Object {
                    "width": undefined,
                  }
                }
              />
              <li
                style={
                  Object {
                    "width": "61%",
                  }
                }
              />
            </ul>
          </div>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="ant-col w-full ant-col-xs-22 ant-col-sm-11 ant-col-lg-8"
    style={Object {}}
  >
    <div
      className="sc-bdfBQB DEPcs flex items-center"
    >
      <div
        className="ant-skeleton ant-skeleton-element ant-skeleton-active my-7"
      >
        <span
          className="ant-skeleton-avatar ant-skeleton-avatar-lg ant-skeleton-avatar-circle"
          style={Object {}}
        />
      </div>
      <div
        className="pl-6 w-full"
      >
        <div
          className="ant-skeleton ant-skeleton-active"
        >
          <div
            className="ant-skeleton-content"
          >
            <h3
              className="ant-skeleton-title"
              style={
                Object {
                  "width": "38%",
                }
              }
            />
            <ul
              className="ant-skeleton-paragraph"
            >
              <li
                style={
                  Object {
                    "width": undefined,
                  }
                }
              />
              <li
                style={
                  Object {
                    "width": undefined,
                  }
                }
              />
              <li
                style={
                  Object {
                    "width": "61%",
                  }
                }
              />
            </ul>
          </div>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="ant-col w-full ant-col-xs-22 ant-col-sm-11 ant-col-lg-8"
    style={Object {}}
  >
    <div
      className="sc-bdfBQB DEPcs flex items-center"
    >
      <div
        className="ant-skeleton ant-skeleton-element ant-skeleton-active my-7"
      >
        <span
          className="ant-skeleton-avatar ant-skeleton-avatar-lg ant-skeleton-avatar-circle"
          style={Object {}}
        />
      </div>
      <div
        className="pl-6 w-full"
      >
        <div
          className="ant-skeleton ant-skeleton-active"
        >
          <div
            className="ant-skeleton-content"
          >
            <h3
              className="ant-skeleton-title"
              style={
                Object {
                  "width": "38%",
                }
              }
            />
            <ul
              className="ant-skeleton-paragraph"
            >
              <li
                style={
                  Object {
                    "width": undefined,
                  }
                }
              />
              <li
                style={
                  Object {
                    "width": undefined,
                  }
                }
              />
              <li
                style={
                  Object {
                    "width": "61%",
                  }
                }
              />
            </ul>
          </div>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="ant-col w-full ant-col-xs-22 ant-col-sm-11 ant-col-lg-8"
    style={Object {}}
  >
    <div
      className="sc-bdfBQB DEPcs flex items-center"
    >
      <div
        className="ant-skeleton ant-skeleton-element ant-skeleton-active my-7"
      >
        <span
          className="ant-skeleton-avatar ant-skeleton-avatar-lg ant-skeleton-avatar-circle"
          style={Object {}}
        />
      </div>
      <div
        className="pl-6 w-full"
      >
        <div
          className="ant-skeleton ant-skeleton-active"
        >
          <div
            className="ant-skeleton-content"
          >
            <h3
              className="ant-skeleton-title"
              style={
                Object {
                  "width": "38%",
                }
              }
            />
            <ul
              className="ant-skeleton-paragraph"
            >
              <li
                style={
                  Object {
                    "width": undefined,
                  }
                }
              />
              <li
                style={
                  Object {
                    "width": undefined,
                  }
                }
              />
              <li
                style={
                  Object {
                    "width": "61%",
                  }
                }
              />
            </ul>
          </div>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="ant-col w-full ant-col-xs-22 ant-col-sm-11 ant-col-lg-8"
    style={Object {}}
  >
    <div
      className="sc-bdfBQB DEPcs flex items-center"
    >
      <div
        className="ant-skeleton ant-skeleton-element ant-skeleton-active my-7"
      >
        <span
          className="ant-skeleton-avatar ant-skeleton-avatar-lg ant-skeleton-avatar-circle"
          style={Object {}}
        />
      </div>
      <div
        className="pl-6 w-full"
      >
        <div
          className="ant-skeleton ant-skeleton-active"
        >
          <div
            className="ant-skeleton-content"
          >
            <h3
              className="ant-skeleton-title"
              style={
                Object {
                  "width": "38%",
                }
              }
            />
            <ul
              className="ant-skeleton-paragraph"
            >
              <li
                style={
                  Object {
                    "width": undefined,
                  }
                }
              />
              <li
                style={
                  Object {
                    "width": undefined,
                  }
                }
              />
              <li
                style={
                  Object {
                    "width": "61%",
                  }
                }
              />
            </ul>
          </div>
        </div>
      </div>
    </div>
  </div>,
]
`;
